src/Types.ts

```diff
@@ -165,11 +171,17 @@ export interface ConditionArgsBase {
   type: ConditionType
 }
 
+export interface TokenArgsForPriceCondition extends TokenArgs {
```

Comment:
it's for lack of a better name. 

I didn't know if we want to include decimals to TokenArgs directly? I thought it was very specific to that priceCondition DSL.

Reply:
@phanpak let's add to TokenArgs directly